PULL_REQUEST_TEMPLATE

```diff
@@ -1,3 +1,6 @@
+##### ABBREVIATED REVIEW BETWEEN 2.11.X AND MASTER IS IN EFFECT
```

Comment:
❓ How about `ABBREVIATED REVIEW PROCESS` to be consistent with the text in the notes below?
❓ Do we need to mention the branch names?